backend/src/main/java/com/ody/notification/service/NotificationService.java

```diff
@@ -84,4 +102,15 @@ public Notification findById(Long notificationId) {
         return notificationRepository.findById(notificationId)
                 .orElseThrow(() -> new OdyNotFoundException("존재하지 않는 알림입니다."));
     }
+
+    @TransactionalEventListener
+    public void unSubscribeTopic(List<Meeting> meetings) {
+        for (Meeting meeting : meetings) {
+            notificationRepository.findAllMeetingIdAndType(meeting.getId(), NotificationType.DEPARTURE_REMINDER)
+                    .forEach(notification -> fcmSubscriber.unSubscribeTopic(
+                            notification.getFcmTopic(),
+                            notification.getMate().getMember().getDeviceToken())
+                    );
```

Comment:
[제안]

토큰을 바로 가져오는 getter 한 개 더 만드는건 어떤가요?
개행도 맞춰주면 어떤가요?

```suggestion
                    .forEach(notification -> fcmSubscriber.unSubscribeTopic(
                                    notification.getFcmTopic(),
                                    notification.getMateMemberDeviceToken()
                            )
                    );
```

Reply:
디미터 법칙 준수하면서 
Mate에서 getMemberDeviceToken() -> Notification에선 getMateDeviceToken()으로 토큰을 가져올 수 있도록 수정했습니다 !

개행도 반영완료 !